rs/tests/idx/colocate_test.rs

```diff
@@ -356,25 +358,82 @@ fn start_test(env: TestEnv, uvm: DeployedUniversalVm) {
     }
 }
 
-fn scp(log: Logger, session: &Session, from: std::path::PathBuf, to: std::path::PathBuf) {
-    let size = fs::metadata(from.clone()).unwrap().len();
+fn fetch_test_dir(env: TestEnv, uvm: &DeployedUniversalVm, session: &Session) {
+    let log = env.logger();
+    let test_dir_tar = Path::new("/home/admin/test.tar");
+    info!(
+        log,
+        "Tarring the test directory on the {UVM_NAME} to {test_dir_tar:?}..."
+    );
+    uvm.block_on_bash_script_from_session(
+        session,
+        &format!("sudo tar -cf {test_dir_tar:?} -C /home/admin/test ."),
```

Comment:
Interesting, why is sudo needed?

Reply:
Because the user in the docker container is `root` meaning that the files it writes to `/home/admin/test` are owned by `root`.